old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -79,44 +81,43 @@ downloadable R script of the entire lesson, available in the footer of each less
 
 
 **Spatial-Temporal Data & Data Management Lesson Series:** This lesson is part
-of a lesson series introducing
-[spatial data and data management in `R` ](https://www.neonscience.org/tutorial/URL).
+of a lesson series introducing 
+#####UPDATE URL BEFORE PUBLISHING#####
```

Comment:
@donal-at-NEON Same as what I did for 04, this URL needs to be updated before publishing.

Reply:
👍